spec/services/libre_translate_spec.rb

```diff
@@ -27,4 +29,38 @@
       expect(described_class.translate_supported?(source, target)).to be true
     end
   end
+
+  describe ".translate" do
+    fab!(:post)
+
+    before do
+      SiteSetting.translator_libretranslate_api_key = api_key
+      Excon
+        .expects(:get)
+        .with(SiteSetting.translator_libretranslate_endpoint + "/languages")
+        .returns(mock_response.new(200, [{ code: "de" }, { code: "en" }].to_json))
```

Comment:
Rather than stubbing `Excon` could you use `stub_request` from core?

https://github.com/discourse/discourse/blob/c815294c57976fd3d12f1114745e6a075d995db9/plugins/discourse-lazy-videos/spec/lib/lazy-videos/lazy_vimeo_spec.rb#L15-L18

Reply:
It's worth a future refactor for consistency, looks like this plugin uses this method heaps instead of just using `stub_request`, maybe it predates `stub_request`